components/timezone/src/ixdtf.rs

```diff
@@ -42,6 +42,8 @@ pub enum ParseError {
     InconsistentTimeZoneOffsets,
     /// There was an invalid Offset.
     InvalidOffsetError,
+    /// There was an invalid subsecond fraction.
+    InvalidFraction,
```

Comment:
I think this is overkill. I'd rather document that sub-nanoseconds are truncated. The formatting code only supports nanosecond precision anyway.